lib/src/realtime/src/connection_state_change.dart

```diff
@@ -35,3 +35,20 @@ class ConnectionStateChange {
     this.retryIn,
   });
 }
+
+/// @nodoc
+@immutable
+class EnrichedConnectionStateChange {
```

Comment:
`EnrichedConnectionStateChange` can extend `ConnectionStateChange` so no need to filter incoming stream messages using `map`